src/components/structures/ThreadPanel.tsx

```diff
@@ -83,7 +83,9 @@ async function getThreadTimelineSet(
         // Filter creation fails if HomeServer does not support the new relation
         // filter fields. We fallback to the threads that have been discovered in
         // the main timeline
-        const timelineSet = new EventTimelineSet(room, {});
+        const timelineSet = new EventTimelineSet(room, {
+            pendingEvents: false,
```

Comment:
does this not break thread-local pending events o_O?

Reply:
yeah, considering renaming this 🧐